Bikeini/components/Filter.js

```diff
@@ -0,0 +1,243 @@
+import React from 'react';
+import {
+  StyleSheet, View, ListView, TouchableOpacity, Text, TextInput,
+} from 'react-native';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import PropTypes from 'prop-types';
+import { changeItems } from '../navigation/actions/FilterActions';
+import ItemCheckbox from './ItemCheckbox';
+
+const styles = StyleSheet.create({
```

Comment:
Idealy we should probably move the stylesheets to an own file but that might be a later problem! 🥶

Reply:
#24 adresses this